### ISSUE TITLE
Object placement control

### ISSUE THREAD

**Issue by YunlianMoon** [allenai/ai2thor#300]
Can I control the placement relationship between objects? for example, mug or cup next to coffee machine， apple next to bread, book put on the desk, etc.

**Comment by winthos** [comment]
We don't currently have support for specifying placement relationship parameters like that in our random spawner. You could try something like use a ton of random seeds and filter out ones that specifically have objects placed close together(ie: only use seeds where the distance from an Apple to Bread is within a threshold).

**Comment by YunlianMoon** [comment]
@winthos 
Do you mean to  change the value of "randomSeed" in  [controller.step(dict(action = 'InitialRandomSpawn', randomSeed = 0)] ？

**Comment by winthos** [comment]
Yes, if you pass in different seeds, the scene will initialize with objects in new random positions based on the seed.
